test/workflow-orchestration.ts

```diff
@@ -49,14 +53,14 @@ describe('When a workflow contains an aggregating step', async function () {
     savedWorkItem.status = WorkItemStatus.SUCCESSFUL;
     savedWorkItem.results = [
       'test/resources/worker-response-sample/catalog0.json',
-      'test/resources/worker-response-sample/catalog1.json',
```

Comment:
I'm not quite following these tests - are we only testing with a single work item and catalog? If so we should modify the tests to test with multiple work items and catalogs.

Reply:
We have two work items (each producing one catalog for one granule, which is what turbo does) followed by an aggregation step.